ambu

```diff
@@ -21,8 +21,18 @@ if [ ! -d "$CURRENTDIR" ]; then
     exit 1
 fi
 
-mkdir -p $BUILDDIR
+command -v ctags >/dev/null 2>&1 || {
+        echo >&2 "ctags required for building your project. Please install it!"
+        exit 1
+}
+
 ctags --language-force=c --exclude=$PROJECTNAME *.ino
+if ! grep -q "$(printf '_TAG_PROGRAM_NAME\tExuberant Ctags\t//')" tags; then
+    echo "ctags generated output not compatible with ambu. Please reinstall it!"
```

Comment:
@Adman This is the correct question to ask I believe. I believe that your comment here should note that it is normally called `excuberant-ctags` in repositories of most distributions, and that there is also a website whose link you might want to attach to this comment too.